packages.yml

```diff
@@ -1,3 +1,3 @@
 packages:
   - package: dbt-labs/dbt_utils
-    version: [">=0.8.0", "<0.9.0"]
+    version: 0.9.5
```

Comment:
Two notes: 
- 0.9.6 is currently latest 
- I would actually prefer you do `version: [">=0.9.0", "<0.10.0"]` as otherwise you can cause version conflicts if the importing project has specified utils 0.9.1 or something. Causes issues like https://github.com/dbt-labs/dbt-core/issues/4775

Reply:
@joellabes Thanks for the feedback! Updated to your suggestion.